docs/reserves-types.md

```diff
@@ -1,4 +1,4 @@
----
+s---
```

Comment:
why is there an s here? haha

Reply:
i guess the kiddo error...ill correct that